app/pages/UcsfDiscoveryForm/UcsfDiscoveryForm.tsx

```diff
@@ -155,18 +158,23 @@ const Page = () => {
       <Section addClass={styles.subtitleMargin} subtitle={stepSubtitle} />
       <div className={styles.eligibilitiesContainer}>
         <div className={styles.refinementsBox}>{refinementsComponent}</div>
-        <div className={styles.navigationButtons}>
-          <Button addClass={styles.goBackBtn} onClick={backToResourceSelection}>
-            Back
-          </Button>
-          <Button
-            addClass={styles.nextBtn}
-            onClick={() => {
-              goToNextStep(selectedResourceSlug);
-            }}
-          >
-            {nextButtonText}
-          </Button>
+        <div className={styles.buttonBarBorder}>
```

Comment:
My reading of the Figma design shows that the button bar should be completely outside of the `.discoveryFormPage`. In the Figma document, you can see that the bar extends to the full width of the browser's viewport. Here, because `.discoveryFormPage` only spans columns 2 through 11 of the page grid, the button bar isn't wide enough. In order to achieve the same layout as the Figma document, I think you need the button bar to be a sibling of the `.grid` (Layout) element.

Reply:
Per our conversation, I've just removed the sticky button bar altogether. That said, I've left in button styling changes that I made to align with Figma design updates 
<img width="1310" alt="image" src="https://user-images.githubusercontent.com/3012520/220768873-790852aa-a106-436b-a6ad-bdc42b04043e.png">